docs/tools/slurm.md

```diff
@@ -20,37 +20,116 @@ The following sections will provide detailed guidance on how to use SLURM to req
 [](){#gh200-slurm}
 ### NVIDIA GH200 GPU Nodes
 
-!!! todo
-    document how slurm can be used on the Grace-Hopper nodes.
-
-    Note how you can link to this section from elsewhere using the anchor above, e.g.:
-
-    ```
-    [using slurm on Grace-Hopper][gh200-slurm]
-    ```
-
-Link to the [Grace-Hopper overview][gh200-node].
-
-An example of using tabs to show srun and sbatch useage to get one GPU per MPI rank:
+The [GH200 nodes on Alps][gh200-node] have four GPUs per node, and SLURM job submissions must be configured appropriately to best make use of the resources.
+Applications that can saturate the GPUs with a single process per GPU should generally prefer this mode.
+[Configuring SLURM jobs to use a single GPU per rank][gh200-slurm-single-rank-per-gpu] is also the most straightforward setup.
+Some applications perform badly with a single rank per GPU, and require use of [NVIDIA's Multi-Process-Service (MPS)](https://docs.nvidia.com/deploy/mps/index.html) to oversubscribe GPUs with multiple ranks per GPU.
+
+The best SLURM configuration is application- and workload-specific, so it is worth testing which works best in your particular case.
+Also see [Scientific Applications][sciapps] for information about recommended application-specific SLURM configurations.
+
+!!! warning
+    The GH200 nodes have their GPUs configured in ["default" compute mode](https://docs.nvidia.com/deploy/mps/index.html#gpu-compute-modes).
+    Unlike "exclusive process" mode, "default" mode allows multiple processes to submit work to a single GPU simultaneously.
```

Comment:
Do we need to make a note that this changed recently on Alps? Maybe not here in the main documentation, but it would be good to have some sort of CHANGELOG for this kind of things.

Maybe it is also worth adding a note on why they are in "default" mode? I think the expectation from users is "exclusive process" (which is mentioned), and it would be good to give a rationale about this choice (i.e. containers not working otherwise).

Reply:
Too short: https://github.com/eth-cscs/cscs-docs/pull/12/commits/2add4041c818cbf356114fadf3e8b3d9dbac22fb?

Reply:
I think it's good enough. If people need/want more information, they can always ask.